benchmarks/monkeypatch_asv.py

```diff
@@ -0,0 +1,29 @@
+from asv_runner.benchmarks.timeraw import TimerawBenchmark, _SeparateProcessTimer
+
+
+# Fix for https://github.com/airspeed-velocity/asv_runner/pull/44
+def _get_timer(self, *param):
+    """
+    Returns a timer that runs the benchmark function in a separate process.
+
+    #### Parameters
+    **param** (`tuple`)
+    : The parameters to pass to the benchmark function.
+
+    #### Returns
+    **timer** (`_SeparateProcessTimer`)
+    : A timer that runs the function in a separate process.
+    """
+    if param:
+
+        def func():
+            # ---------- OUR CHANGES: ADDED RETURN STATEMENT ----------
+            return self.func(*param)
```

Comment:
Another bug I had to fix from outside was that this function was missing a return statement